### ISSUE TITLE
release:prepare fail: Unable to commit files

### ISSUE THREAD

**Issue by mariuszs** [takari/takari-lifecycle#12]
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-release-plugin:2.5.1:prepare (default-cli) on project catch-exception-parent: Unable to commit files
[ERROR] Provider message:
[ERROR] The git-add command failed.
[ERROR] Command output:
[ERROR] fatal: pathspec '.polyglot.pom.groovy' did not match any files
[ERROR] -> [Help 1]
```

**Comment by mariuszs** [comment]
Is this possible to release takari-jar to sonatype oss?

**Comment by jvanzyl** [comment]
Yes, we do all the time. Are you trying to release a project using the Takari Lifecycle that is also using Polyglot for Maven?

**Comment by mariuszs** [comment]
Yes. My project https://github.com/mariuszs/catch-exception/blob/2.x/pom.groovy

Are you using maven-release-plugin for releasing?

I see that mvn deploy works (https://oss.sonatype.org/content/repositories/releases/eu/codearte/catch-exception/catch-throwable/2.0.0-ALPHA-1/catch-throwable-2.0.0-ALPHA-1.jar)

How to manage version change with polyglot? mvn versions:set is not working.

**Comment by jvanzyl** [comment]
I'll take a look this weekend. You are officially the first person in the world to ever try to use the release plugin with a Polyglot Maven project :-) We'll get it working.

**Comment by mikaelhg** [comment]
Did you ever get it working, since I'm hitting the bug with polyglot 0.4.0 YAML in 2019?

**Comment by mosabua** [comment]
Nobody ever worked on release plugin support for polyglot maven. Maybe we should close this issue.

**Comment by mikaelhg** [comment]
The reality is that the release plugin hasn't been updated since 2.5 in 2014 and 2.5.3 in 2015, and a lot of folks have simply dropped it from the release process. That's the approach I took.
